includes/bluespice-data/settings.d/099-Custom.php

```diff
@@ -7,8 +7,8 @@
 $wgMemCachedServers = [ "127.0.0.1:11211" ];
 wfLoadExtension( 'BlueSpiceExtendedSearch' );
 $GLOBALS['wgSearchType'] = 'BS\\ExtendedSearch\\MediaWiki\\Backend\\BlueSpiceSearch';
-wfLoadExtension( 'BlueSpicePrivacy' );
-wfLoadExtension( "BlueSpiceUEModulePDF" );
+// wfLoadExtension( 'BlueSpicePrivacy' );
+// wfLoadExtension( "BlueSpiceUEModulePDF" );
```

Comment:
Why would they be deactivated? What is this file even for? The Docker container is meant to provide the infrastructure to run those extensions.

Reply:
yea will uncomment it, this was due to the BlueSpiceUEModulePDF was having some issues from the bluespice repo and after discussing it with dejan i commented it and same for BlueSpicePrivacy